plugins/RocketsPlugin/Timeout.h

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2015-2018, EPFL/Blue Brain Project
+ *
+ * Responsible Author: Daniel.Nachbaur@epfl.ch
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+#include <condition_variable>
+#include <functional>
+#include <future>
+#include <mutex>
+
+namespace brayns
+{
+/**
+ * Implements the setTimeout() and clearTimeout() mechanics from Javascript
+ * (https://www.w3schools.com/jsref/met_win_settimeout.asp), following the same
```

Comment:
[setTimeout()](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) is a better link to the doc 😄

Reply:
Ok, thanks :smile: